gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/persistence/BackupManager.java

```diff
@@ -165,8 +164,29 @@ private File checkBaseline(File baselineParentDir) throws IOException{
     
     return baselineDir;
   }
-  
-  public HashSet<PersistentID> finishBackup(File targetDir, File baselineDir) throws IOException {
+
+  private Collection<DiskStoreImpl> getDiskStoresToBackUp(byte diskStoresToBackup) {
+    Collection<DiskStoreImpl> allDses = cache.listDiskStoresIncludingRegionOwned();
+    switch (diskStoresToBackup) {
+      case FinishBackupRequest.DISKSTORE_ALL:
+        return allDses;
+
+      case FinishBackupRequest.DISKSTORE_ALL_BUT_DD:
+        allDses.removeIf(ds -> ds.getName().equals("GFXD-DD-DISKSTORE"));
+        return allDses;
+
+      case FinishBackupRequest.DISKSTORE_DD:
+        Optional<DiskStoreImpl> ods = allDses.stream().filter
+            (ds -> ds.getName().equals("GFXD-DD-DISKSTORE")).findFirst();
```

Comment:
Use GFXD_DD_DISKSTORE_NAME from GfXDConstant.

Reply:
This is gemfire code. GfxdConstants is not visible here, hence it is like this.

Reply:
ok.